src/Microsoft.AspNetCore.Authentication.Abstractions/IAuthenticationService.cs

```diff
@@ -12,9 +12,6 @@ public interface IAuthenticationService
         Task<AuthenticateResult> AuthenticateAsync(HttpContext context, string scheme);
         Task ChallengeAsync(HttpContext context, string scheme, AuthenticationProperties properties, ChallengeBehavior behavior);
         Task ForbidAsync(HttpContext context, string scheme, AuthenticationProperties properties);
```

Comment:
Until we sort out Challenge vs Forbid, Forbid is not a 1st class thing, it's a subset of Challenge.